doc/tutorial/alphabet/index.md

```diff
@@ -0,0 +1,329 @@
+\snippet alphabet_main.cpp rank
+
+<br/>
+
+## The char representation of an alphabet symbol
+
+Our alphabets also have a character representation, because it is more intuitive to work
+with them than using the rank. Each alphabet symbol is represented by its respective character
+whenever possible (`A ⟼ 'A'`). Analogously to the rank, SeqAn provides the function 
+seqan3::to_char for converting a symbol to its character representation.
+
+\snippet alphabet_main.cpp char
+
+Above you have seen that you can assign an alphabet symbol from a character with seqan3::from_char. 
+In contrast to the rank interface, this assignment is not a bijection, because the whole spectrum of available
+chars is mapped to values inside the alphabet. For instance, assigning to seqan3::dna4 from any character other
+than `C`, `G` or `T` results in the value 'A'_dna4 and assigning from any character except `A`, `C`, `G` or `T`
+to seqan3::dna5 results in the value 'N'_dna5. You can avoid the implicit conversion by using
+seqan3::assign_char_strict, which throws seqan3::invalid_char_assignment on invalid characters.
+
+\snippet alphabet_main.cpp char_strict
+
+<br/>
+
+## Obtaining the alphabet size
+You can retrieve the alphabet size by accessing the class member variable `value_size`,
+which is implemented in all seqan3::Alphabet instances.
+\snippet alphabet_main.cpp size
+
+<br/>
+
+---
+
+## Containers over alphabets
+
+In SeqAn you can use the STL containers to model e.g. sequences, sets or mappings with our alphabets.
+The following example shows some exemplary contexts for their use. 
+For **sequences** we recommend the std::vector with one of SeqAn's alphabet types. 
+Please note how easily a sequence can be created via the string literal.
+\snippet alphabet_main.cpp containers
+
+<br/>
+
+---
+
+## Example
+
+To wrap up this section on the nucleotide alphabet, the following exercise will let you 
+practise the use of a SeqAn alphabet and its related functions.
+It will also show you a handy advantage of using a vector over an alphabet instead 
+of using `std::string`: The rank representation can be used straight as an array
+index (opposed to e.g. using a map with logarithmic access times).
+
+\assignment{Excercise: GC content of a sequence}
+An important property of DNA and RNA molecules is the *GC content*,
+which is the percentage of nucleobases that are either Guanine or Cytosine.
+Given the nucleotide counts \f$n_A\f$, \f$n_T\f$, \f$n_G\f$, \f$n_C\f$ the GC content \f$c\f$ is calculated as
+\f[ c = \frac{n_G + n_C}{n_A + n_T + n_G + n_C} \f]
+Write a program that
+1. reads a sequence as command line argument into a vector of seqan3::dna5,
+2. counts the number of occurrences for each nucleotide in an array of size `alphabet size` and
+3. calculates the GC content.
+
+The seqan3::dna5 type ensures that invalid characters in the input sequence are converted to 'N'.
+Note that these characters should not influence the GC content.
+\endassignment
```

Comment:
Add: `Check your program by executing ./bla ACGATCAGCTACGC , it should print blubb` to the assignment so they have a reference.